python/cudf/cudf/tests/test_string.py

```diff
@@ -2143,6 +2143,42 @@ def test_string_str_rindex(data, sub, er):
         assert not er
 
 
+@pytest.mark.parametrize(
+    "data,sub",
+    [
+        (
+            ["abc", "xyz", "a", "ab", "123", "097"],
+            ["b", "y", "a", "c", "4", "8"],
+        ),
+        (
+            ["A B", "1.5", "3,000", "23", "³", "⅕"],
+            ["A B", ".", ",", "1", " ", " "],
+        ),
+        (
+            [" ", "\t", "\r", "\f ", "\n", ""],
+            ["", "\t", "\r", "xx", "yy", "zz"],
+        ),
+        (
+            ["$", "B", "Aab$", "$$ca", "C$B$", "cat"],
+            ["$", "B", "ab", "*", "@", "dog"],
+        ),
+        (
+            ["hello", "there", "world", "-1234", "", "accént"],
+            ["lo", "e", "o", "+1234", " ", "e"],
+        ),
+        (
+            ["1. Ant.  ", "2. Bee!\n", "3. Cat?\t", "", "x", "é"],
+            ["A", "B", "C", " ", "y", "e"],
+        ),
+    ],
+)
+def test_string_contains_multi(data, sub):
+    gs = Series(data)
```

Comment:
Don't see any null tests, it would be nice to have one.

Reply:
Added nulls to two tests in the dataset.